tabular/src/autogluon/tabular/models/lgb/callbacks.py

```diff
@@ -69,8 +29,9 @@ def early_stopping_custom(stopping_rounds, first_metric_only=False, metrics_to_u
 
     Parameters
     ----------
-    stopping_rounds : int
-       The possible number of rounds without the trend occurrence.
+    stopping_rounds : int or tuple
+       If int, The possible number of rounds without the trend occurrence.
```

Comment:
"trend occurrence" should be explained in more detail

Reply:
Would prefer not to edit since this is copy pasted from LightGBM's early stopping function.